code/clue.js

```diff
@@ -1,59 +1,202 @@
   lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
-  age: 45,
+  color: '#556B2F',
+  age: 55,
+  occupation: 'Entrepreneur',
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: '#800080',
+  age: 46,
+  occupation: 'Scientist',
+  image: 'assets/plum.png'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlett',
+  color: '#800000',
+  age: 40,
+  occupation: 'Actress',
+  image: 'assets/scarlet.png'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#00008B',
+  age: 49,
+  occupation: 'Ornithologist',
+  image: 'assets/peacock.png'
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: '#FFA500',
+  age: 72,
+  occupation: 'Hunter',
+  image: 'assets/mustard.png'
+}
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: '#FDF5E6',
+  age: 70,
+  occupation: 'Housekeeper',
+  image: 'assets/white.png'
+  }
+
+
+// OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
-  name: 'rope',
+  name: 'Rope',
+  weight: 8
+}
+
+const knife = {
+  name: 'Knife',
+  weight: 2
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 4
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 8,
+}
+
+const poison = {
+name: 'Poison',
+weight: 1
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 12
+}
+
+const bat = {
+  name: 'Bat',
   weight: 10
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const trophy = {
+  name: 'Trophy',
+  weight: 8
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 8
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// ARRAYS FOR ALL SUSPECTS, WEAPONS AND ROOMS
 
 const suspects = [
   mrGreen,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
+]
+
 
-const rooms = []
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// AN OBJECT THAT KEEPS THE MYSTERY.
 
-// This function will be invoked when you click on the killer card.
+const mystery = {
+  killer: null,
+  weapon: null,
+  room: null
```

Comment:
Great that you set the value to null to keep from exposing the mystery before all cards are clicked.